pkg/controller/chaosengine/chaosengine_controller.go

```diff
@@ -171,7 +171,7 @@ func (r *ReconcileChaosEngine) Reconcile(request reconcile.Request) (reconcile.R
 		// Determine whether apps with matching labels have chaos annotation set to true
 		engine, err = resource.CheckChaosAnnotation(engine)
 		if err != nil {
-			chaosTypes.Log.Info("Annotation check failed with error: ", err)
+			chaosTypes.Log.Info("Annotation check failed with", "error:", err)
```

Comment:
This will fix the issue below:
```
{"level":"dpanic","ts":1577944967.5595882,"logger":"controller_chaosengine","msg":"odd number of arguments passed as key-value pairs for logging","ignored key":"resource type 'deployment', err: no chaos-candidate found","stacktrace":"github.com/litmuschaos/chaos-operator/vendor/github.com/go-logr/zapr.handleFields\n\tsrc/github.com/litmuschaos/chaos-operator/vendor/github.com/go-logr/zapr/zapr.go:106\ngithub.com/litmuschaos/chaos-operator/vendor/github.com/go-logr/zapr.(*infoLogger).Info\n\tsrc/github.com/litmuschaos/chaos-operator/vendor/github.com/go-logr/zapr/zapr.go:70\ngithub.com/litmuschaos/chaos-operator/pkg/controller/chaosengine.(*ReconcileChaosEngine).Reconcile\n\tsrc/github.com/litmuschaos/chaos-operator/pkg/controller/chaosengine/chaosengine_controller.go:174\ngithub.com/litmuschaos/chaos-operator/vendor/sigs.k8s.io/controller-runtime/pkg/internal/controller.(*Controller).reconcileHandler\n\tsrc/github.com/litmuschaos/chaos-operator/vendor/sigs.k8s.io/controller-runtime/pkg/internal/controller/controller.go:216\ngithub.com/litmuschaos/chaos-operator/vendor/sigs.k8s.io/controller-runtime/pkg/internal/controller.(*Controller).processNextWorkItem\n\tsrc/github.com/litmuschaos/chaos-operator/vendor/sigs.k8s.io/controller-runtime/pkg/internal/controller/controller.go:192\ngithub.com/litmuschaos/chaos-operator/vendor/sigs.k8s.io/controller-runtime/pkg/internal/controller.(*Controller).worker\n\tsrc/github.com/litmuschaos/chaos-operator/vendor/sigs.k8s.io/controller-runtime/pkg/internal/controller/controller.go:171\ngithub.com/litmuschaos/chaos-operator/vendor/k8s.io/apimachinery/pkg/util/wait.JitterUntil.func1\n\tsrc/github.com/litmuschaos/chaos-operator/vendor/k8s.io/apimachinery/pkg/util/wait/wait.go:152\ngithub.com/litmuschaos/chaos-operator/vendor/k8s.io/apimachinery/pkg/util/wait.JitterUntil\n\tsrc/github.com/litmuschaos/chaos-operator/vendor/k8s.io/apimachinery/pkg/util/wait/wait.go:153\ngithub.com/litmuschaos/chaos-operator/vendor/k8s.io/apimachinery/pkg/util/wait.Until\n\tsrc/github.com/litmuschaos/chaos-operator/vendor/k8s.io/apimachinery/pkg/util/wait/wait.go:88"}
{"level":"info","ts":1577944967.5595794,"logger":"controller_chaosengine","msg":"Annotation check failed with error: "}
```